### ISSUE TITLE
Transforming EPSG:4978 geocent projection fails?

### ISSUE THREAD

**Issue by jwgmeligmeyling** [proj4js/proj4js#195]
I am trying to transform points to and form the EPSG:4978 projection. I have found multiple definitions on the internet, but none of them seem to work. For example the empty definition: (source: http://spatialreference.org/ref/epsg/wgs-84-2/proj4js/)

``` js
proj4.defs("EPSG:4978", "");
```

Throws an error.

The defintion:

``` js
proj4.defs("EPSG:4978", "+proj=geocent +datum=WGS84 +units=m +no_defs");
```

Can be created, but transforming points fails with an error. For example:

``` js
proj4(proj4.defs("EPSG:4978"), proj4.defs("EPSG:4978"), [2000000, 600000]);
```

Then I found this issue on a mailing list: http://lists.osgeo.org/pipermail/metacrs/2010-September/000490.html

And this unanswered question on the GIS stackexchange: http://gis.stackexchange.com/questions/181929/proj4js-with-geocentric-crs

I am wondering, why is this failing? Is this a known issue? Am I using an older version of proj4js?

**Comment by calvinmetcalf** [comment]
Thanks not a supported projection type

**Comment by JWGmeligMeyling** [comment]
Aha, how can one impkement it?

**Comment by calvinmetcalf** [comment]
#244 is probably a good guide, adding geocentric would probably be easier as there really aren't any parameters, might be a little tricky just because no projections currently require a z

**Comment by codingdna2** [comment]
Can someone provide a simple example? I'm not able to use the workaround. I want to convert from ESPG:4326 to EPSG:4328 but I'm getting error with any system which uses +proj=geocent

**Comment by ahocevar** [comment]
Fixed with #327.
